cp1404practicals/prac_05/emails.py

```diff
@@ -0,0 +1,23 @@
+"""
+CP1404 Practical 05
+emails
+"""
+
+email = input("Email: ")
+email_book = {}
+while email != "":
+    temp_name = email.split("@")[0]  # splits email at "@" and takes the first part
+    temp_name = temp_name.split(".")
+    persons_name = ""
+    for name in temp_name:
+        persons_name += name.title() + " "
+    try:
+        name_change = input("Is your name {}? (Y/n) ".format(persons_name))
+        if name_change[0] in "Nn":
```

Comment:
You should use the .lower() method to convert the input to lowercase and then use the comparison operator (==) to check if name_change is equal to "n".